frontend/src/metabase/models/containers/ModelDetailPage/ModelDetailPage.unit.spec.tsx

```diff
@@ -465,6 +465,28 @@ describe("ModelDetailPage", () => {
           ).toBeInTheDocument();
         });
 
+        it("lists existing public query actions with public label", async () => {
+          const model = getModel();
+          const action = createMockQueryAction({
+            model_id: model.id(),
+            public_uuid: "mock-uuid",
+          });
+          await setupActions({ model, actions: [action] });
+
+          expect(screen.getByText(action.name)).toBeInTheDocument();
+          expect(screen.getByText(TEST_QUERY)).toBeInTheDocument();
+          expect(screen.getByText(/Public Action/)).toBeInTheDocument();
+          const expectedSeparator = "·";
+          expect(
+            screen.getByText(new RegExp(expectedSeparator)),
+          ).toBeInTheDocument();
```

Comment:
I think testing the separator is a bit too excessive. I'd hate a test to fail if we decide to change it

Reply:
yeah sounds fair 👍